cli/cmd/stat_summary.go

```diff
@@ -0,0 +1,254 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/runconduit/conduit/controller/api/util"
+	pb "github.com/runconduit/conduit/controller/gen/public"
+	"github.com/spf13/cobra"
+)
+
+var namespace, resourceType, resourceName string
+var outToNamespace, outToType, outToName string
+var outFromNamespace, outFromType, outFromName string
+
+var statSummaryCommand = &cobra.Command{
+	Use:   "statsummary [flags] deployment [RESOURCE]",
+	Short: "Display traffic stats about one or many resources",
+	Long: `Display traffic stats about one or many resources.
+
+Valid resource types include:
+
+	* deployment
+
+This command will hide resources that have completed, such as pods that are in the Succeeded or Failed phases.`,
+	Example: `  conduit statsummary deployments hello1 -a test `,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		switch len(args) {
+		case 1:
+			resourceType = args[0]
+		case 2:
+			resourceType = args[0]
+			resourceName = args[1]
+		default:
+			return errors.New("please specify one resource only")
+		}
+
+		client, err := newPublicAPIClient()
+		if err != nil {
+			return fmt.Errorf("error creating api client while making stats request: %v", err)
+		}
+
+		output, err := requestStatSummaryFromAPI(client)
+		if err != nil {
+			return err
+		}
+
+		_, err = fmt.Print(output)
+
+		return err
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(statSummaryCommand)
+	addControlPlaneNetworkingArgs(statSummaryCommand)
+	// TODO: the -n flag is taken up by conduit-namespace :( we should move it to something else so this can have -n
+	statSummaryCommand.PersistentFlags().StringVarP(&namespace, "namespace", "a", "default", "namespace of the specified resource")
+	statSummaryCommand.PersistentFlags().StringVarP(&timeWindow, "time-window", "t", "1m", "Stat window.  One of: '10s', '1m', '10m', '1h'.")
```

Comment:
I think that would be really nice, since we're not doing any aggregations, but to avoid hashing out apis within this PR, I'll leave as is. Should be an easy change in a follow up!  Made #686 to track.